gnovm/cmd/gno/util.go

```diff
@@ -100,6 +100,64 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 	return paths, nil
 }
 
+func gnoPackagePathsFromPattern(patterns []string) ([]string, error) {
+	paths := []string{}
+	recursiveLookup := false
+	for _, p := range patterns {
+		// TODO:
+		//     - Support more patterns
```

Comment:
Just curious, what kinds of other patterns would you support?

Reply:
See patterns supported by `go test` command
```
go test path/to/dir
go test path/to/dir/...
go test ./...
go test path/.../dir
go test ./.../test
go test package/import/path
...
```